Translation/settings.py

```diff
@@ -209,7 +210,7 @@
 MEDIA_URL = '/media/'
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media/')
 STATIC_ROOT = os.path.join(BASE_DIR, "static/")
-PRINTED_DRAFT_TRANSLATIONS_ROOT = os.path.join(BASE_DIR, 'draft_translations/')
+PRINTED_DRAFT_TRANSLATIONS_ROOT = os.path.join(MEDIA_ROOT, 'draft_translations/')
```

Comment:
Why?

As comparison, `/final_pdf/` is at `BASE_DIR`.

Reply:
Ah, I was going to move that one also. I suppose *.pdf should be a good enough filter for the file path field.